.github/workflows/go-update.yml

```diff
@@ -0,0 +1,79 @@
+name: Open Go Update PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      go_version:
+        description: 'Go version'
+        required: true
+        type: string
+      open_pr:
+        description: 'Whether to a PR'
+        required: false
+        type: boolean
+        default: true
+      draft:
+        description: 'Open a Draft PR'
+        required: false
+        type: boolean
+        default: false
+      branch:
+        description: 'Git branch to use (defaults to "gobot/go-update-<go_version>")'
+        required: false
+        type: string
+        default: ''
+
+env:
+  BRANCH_NAME: ${{ inputs.branch || format('gobot/go-update-{0})', inputs.go_version) }}
+
+jobs:
```

Comment:
should we add a `if` condition here to restrict the permissions to trigger this workflow (for instance to a list of hardcoded `github.actor`, not sure we can dynamically target a team)

Reply:
I believe workflows can only be triggered by someone having `write` access to the repo, so if someone who's not supposed to can trigger it we have a big issue already :smile:

Reply:
I checked and there is an action to check team membership, the question now is do we want to ?
I don't think we need to as workflows need write permission to be triggered anyway

Reply:
I agree it's an additional verification. However, you maybe don't want anyone having write permission to trigger it :)

Reply:
> However, you maybe don't want anyone having write permission to trigger it :)

That's a good remark, but considering the created PR would need reviews anyway, and you can do the exact same changes manually if you'd like, I would say it's not needed.